beacon-chain/Dockerfile

```diff
@@ -20,6 +20,6 @@ ENV JAVA_OPTS="-Xmx4g"
 # API port: https://docs.teku.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#rest-api-port
 EXPOSE $BEACON_API_PORT
 # P2P port: https://docs.teku.consensys.net/en/latest/Reference/CLI/CLI-Syntax/#p2p-port
-#EXPOSE 9000
+EXPOSE $P2P_PORT
```

Comment:
we don't normally do this ever because we do it in the compose file, we used to do it way back before we had to hardcode the port. it worked and does work fine and better using the compose file